extension/ts/main/extensions/author/authorPage/pull.ts

```diff
@@ -57,7 +58,7 @@ const finishPull = (pullData: PullData) => {
 const onPull = async () =>
 	loaderOverlaid(async (): Promise<PullData> => {
 		const {uuid, name} = getAuthorInfo();
-		const [author, books] = await Promise.all([Author.getAuthor(uuid), Book.getBooks({author: uuid})]);
+		const [author, books] = await Promise.all([Author.getAuthor(uuid), getAuthorPageBooks(uuid)]);
```

Comment:
super nit: why doesn't this live in `adapters/Book` like the other one?

Reply:
The rationale was that it depends on the specific markup of author pages, not on book pages, so these APIs are meant to reflect the structure of library thing. Once upon a time there was only `getBooks` and no `getBooksFromSomeSpecificLibraryThingPage` so it made sense.

To be honest WHOLE thing is weird and needs a restructure. Might be worth adding this to #133

Reply:
But for now I will move it to the adapter

Reply:
OK I started moving it, but the problem is that this function _only works_ if invoked ON the author page. What needs to happen is it works when invoked form any page and then it can safely move into the adapter